conda_setup.py

```diff
@@ -6,10 +6,9 @@
 from Cython.Build import cythonize
 
 extensions = [Extension('pyscipopt.scip', ['pyscipopt/scip.pyx'],
-                        extra_compile_args=['-UNDEBUG'],
```

Comment:
What's the reason for this? I don't remember why I put it there, but I assume that it was necessary.